README.md

```diff
@@ -27,20 +27,20 @@ $ git clone --depth=1 https://github.com/juristr/webpack-typescript-starter.git
 ```
 Maybe remove the `.git` directory and start with a fresh one.
 
-run `npm i` to install all dependencies.
+run `$ npm i` to install all dependencies.
 
 ### available commands (npm run xxx):
```

Comment:
Or better even, maybe we could structure this part as they do it on the AngularClass webpack starter: https://github.com/AngularClass/angular2-webpack-starter#quick-start

What do you think? I quite like it to be honest